lib/pf/mfa/Akamai.pm

```diff
@@ -151,32 +152,43 @@ sub check_user {
             return $ACTIONS{'push'}->($self,$default_device[0]->{'device'},$username);
        }
     }
-    elsif ($self->radius_mfa_method eq 'strip-otp' || $self->radius_mfa_method eq 'second-password') {
+    elsif ($self->radius_mfa_method eq 'strip-otp' || $self->radius_mfa_method eq 'second-password' || $self->radius_mfa_method eq 'sms' || $self->radius_mfa_method eq 'phone') {
```

Comment:
I think that this `if` is starting to get a bit too fat (insert pun about you know who here)

A lookup in a map would make things simpler/clearer

Reply:
It looks like just an ```else``` would do since that is the list of all the other choices.